compose/django/Dockerfile

```diff
@@ -112,56 +112,8 @@ WORKDIR ${APP_HOME}/backend
 
 ENTRYPOINT ["/entrypoint"]
 
-# Production stages
-FROM node:22-slim AS frontend-build-stage
-ARG VITE_HOCUSPOCUS_URL
-ARG VITE_BACKEND_URL
-ARG VITE_BACKEND_WS_URL
-ARG SENTRY_AUTH_TOKEN
-ARG BUILD_ENVIRONMENT
-ARG GIT_REVISION
-ARG SENTRY_DSN
-ENV VITE_HOCUSPOCUS_URL=${VITE_HOCUSPOCUS_URL}
-ENV VITE_BACKEND_URL=${VITE_BACKEND_URL}
-ENV VITE_BACKEND_WS_URL=${VITE_BACKEND_WS_URL}
-ENV SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN}
-ENV VITE_SENTRY_RELEASE=${GIT_REVISION}
-ENV SENTRY_RELEASE=${GIT_REVISION}
-ENV VITE_SENTRY_DSN=${SENTRY_DSN}
-
-RUN corepack enable
-
-# Install certs
-RUN apt-get update && apt-get install --no-install-recommends -y \
-  ca-certificates \
-  python3 \
-  build-essential \
-  pkg-config \
-  libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev \
-  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
-  && rm -rf /var/lib/apt/lists/*
-
-RUN mkdir -p /frontend/app
-
-WORKDIR /frontend
-
-COPY ./frontend/app/package.json ./app
-COPY ./frontend/pnpm-lock.yaml ./
-COPY ./frontend/pnpm-workspace.yaml ./
-
-RUN pnpm install --frozen-lockfile --strict-peer-dependencies
-
-# pnpm install doesn't work with NODE_ENV=production somehow.
-ENV NODE_ENV=${BUILD_ENVIRONMENT}
-
-WORKDIR /frontend/app
-
-COPY ./frontend/app .
-
-RUN ./node_modules/.bin/vite build
-
 FROM python-run-stage AS python-run-stage-prod
 ARG GIT_REVISION
 ENV SENTRY_RELEASE=${GIT_REVISION}
-COPY --from=frontend-build-stage /frontend/app/dist/static/ /app/backend/coordnet/static/
```

Comment:
You assume here that the production deployment will be using the docker compose namespace.
As far as I can tell, there is no way to make the fly.io build work without the build stages being within the same Dockerfile (or making the other image available somewhere I suppose).
See fly.io docs: https://fly.io/docs/reference/configuration/#the-build-section